src/bankid.ts

```diff
@@ -487,4 +784,20 @@ export class BankIdClientV6 extends BankIdClient {
   async collect(parameters: CollectRequest) {
     return super.collect(parameters) as Promise<CollectResponseV6>;
   }
+
+  async payment(parameters: PaymentRequest): Promise<PaymentResponse> {
+    return super.payment(parameters);
+  }
+
+  async phoneAuth(parameters: PhoneAuthRequest): Promise<PhoneAuthResponse> {
+    return super.phoneAuth(parameters);
+  }
+
+  async phoneSign(parameters: PhoneSignRequest): Promise<PhoneSignResponse> {
+    return super.phoneSign(parameters);
+  }
+
+  async otherPayment(parameters: OtherPaymentRequest): Promise<OtherPaymentResponse> {
+    return super.otherPayment(parameters);
+  }
```

Comment:
Since we are not changing the implementation from base this feels a bit noisy? The other overrides we have in this class are due to changes in behavior or return types. I’m 👍 with removing them.

```suggestion
```